src/Linter/LintingException.php

```diff
@@ -14,7 +14,50 @@
 
 /**
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * @internal
```

Comment:
Needed to skip `ProjectCodeTest::testThatSrcClassesNotAbuseInterfaces`.

I think there is no need for interfaces here.